src/flb_metrics_exporter.c

```diff
@@ -299,6 +299,15 @@ struct cmt *flb_me_get_cmetrics(struct flb_config *ctx)
         }
     }
 
+    if (ctx->log != NULL) {
+        ret = cmt_cat(cmt, ctx->log->metrics->cmt);
+        if (ret == -1) {
```

Comment:
Invert this conditional son instead of matching `-1` it matches anything different than `0`.

The idea is that while at the moment this function returns `-1` in every error return branch this is not great if we care about diagnosing issues and we'd like to leave the door open to improve it in the future.

Reply:
I fully agree with you and I've updated this.

I had actually asked about this pattern a few weeks ago and got conflicting advice and was basically told "this is how it's done". Another example of something that would be good to codify in CONTRIBUTING.md.